kevlar/tests/test_gentrio.py

```diff
@@ -111,16 +111,19 @@ def test_gen_muts():
 
     testrefrs = [
         'T', 'A', 'GAGAGGGTTACATACGCAGAAAAAGACACACGCTACTGCCCCGCATAGCC', 'A',
-        'C', 'A', 'A', 'C', 'CCAATTAACGTGGAAGCTCGGGGTACCAGCGGACTTAGTGTTCCCGTCT'
-        'TCGGGTTTAATACTATACAGGCCGCAGATACGGCGAACGACGACCCCTTAATTAGACGTGAATTCACTT'
-        'TAGCATCCTTGCTCCTTCCGAAAGACCCCTATGGAAGCAACTCCCGAAACCCCGAAAGGT', 'C'
+        'C', 'A', 'AGTTTGTCGACCTCGAACTTGCCGCC', 'A', 'T', 'C'
```

Comment:
Most of these changes to the `test_gentrio.py` file are to account for additional invocations of the random number generator while simulating insertions.